_events/sr2020/southampton-tech-day-january.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Southampton February Tech Day
+date: 2020-02-08 10:00:00
+layout: event
+type: techday
+location: Southampton University
+---
+
+Tech Days are opportunities for teams to spend a whole day working on their
+robot with lots of help available. They're also an opportunity to see how other
+teams are doing or get more direct help with your robots.
+
+We provide a space for you to work in, with power and internet access, as well
+as volunteers able to help you with your kits and hands-on guidance with your
+robots.
+
```

Comment:
Might be worth including the link to the signup form? Not sure if we care about that for the Southampton ones though.

Reply:
I think it's still an important thing to capture. Running a tech day no one turns up to is equally possible.